src/components/TokenSelect/TokenOptions.tsx

```diff
@@ -219,7 +219,7 @@ const TokenOptions = forwardRef<TokenOptionsHandle, TokenOptionsProps>(function
       onBlur={onBlur}
       onFocus={onFocus}
       onMouseMove={onMouseMove}
-      style={{ overflow: 'hidden', minHeight: 400 }}
+      style={{ overflow: 'hidden', minWidth: 400, minHeight: Math.min(tokens.length, 6) * ITEM_SIZE }}
```

Comment:
also needed to adjust the sizing of the modal content

Reply:
oo i would move the `6` to a constant